crates/metering/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "base-reth-metering"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description = "Transaction Metering RPC Support"
+
+[lints]
+workspace = true
+
+[dependencies]
+# reth
+reth.workspace = true
+reth-provider.workspace = true
+reth-primitives.workspace = true
+reth-evm.workspace = true
+reth-optimism-evm.workspace = true
+reth-optimism-chainspec.workspace = true
+
+# alloy
+alloy-primitives.workspace = true
+alloy-consensus.workspace = true
+alloy-eips.workspace = true
+
+# op-alloy
+op-alloy-consensus.workspace = true
+
+# revm
+revm.workspace = true
+
+# rpc
+jsonrpsee.workspace = true
+
+# async
+tokio.workspace = true
+
+# misc
+tracing.workspace = true
+serde.workspace = true
+serde_json.workspace = true
+eyre.workspace = true
+hex = "0.4"
```

Comment:
nit: can we move the version for this into the root Cargo.toml?

Reply:
removed in a9574f071702ef142a2860875e4c21dc6a3f8400